#
# old_revision [c1f6433718c1bf5a1b8345693c3e088063af3cd2]
#
# patch "configure.pl"
#  from [a87f68a62bf7114de8146d021ec1003dbe2e3ff5]
#    to [5d6e3060c564f114e16f6a4d207fa8f145ed0610]
# 
# patch "src/build-data/makefile/unix.in"
#  from [642755d569ece07a23b635219a4d8fd62a4fd710]
#    to [0a38c397f55fdb5df95b7750fa3895b44aa4be60]
# 
# patch "src/build-data/makefile/unix_shr.in"
#  from [99562e4eb73592c98a2e41842e4e5f2342aff7b7]
#    to [5c2c8418a24ade03d2a66eb5ab677907c246864e]
#
============================================================
--- configure.pl	a87f68a62bf7114de8146d021ec1003dbe2e3ff5
+++ configure.pl	5d6e3060c564f114e16f6a4d207fa8f145ed0610
@@ -205,7 +205,7 @@ sub main {
 
     process_template(File::Spec->catfile(
                          $$config{'config-dir'}, 'botan.doxy.in'),
-                     File::Spec->catfile($$config{'doc_src_dir'}, 'botan.doxy'),
+                     File::Spec->catfile($$config{'build_dir'}, 'botan.doxy'),
                      $config);
 
     $$config{'includes'}{'build.h'} = $$config{'build_dir'};
============================================================
--- src/build-data/makefile/unix.in	642755d569ece07a23b635219a4d8fd62a4fd710
+++ src/build-data/makefile/unix.in	0a38c397f55fdb5df95b7750fa3895b44aa4be60
@@ -92,7 +92,7 @@ doxygen:
 static: $(STATIC_LIB)
 
 doxygen:
-	doxygen %{doc_src_dir}/botan.doxy
+	doxygen %{build_dir}/botan.doxy
 
 clean:
 	$(RM_R) %{build_dir}/lib/* %{build_dir}/checks/*
@@ -100,7 +100,7 @@ distclean: clean
 
 distclean: clean
 	$(RM_R) %{build_dir}
-	$(RM_R) %{doc_src_dir}/doxygen %{doc_src_dir}/botan.doxy
+	$(RM_R) %{doc_src_dir}/doxygen
 	$(RM) Makefile $(CONFIG_SCRIPT) $(PKGCONFIG)
 
 install: $(LIBRARIES)
============================================================
--- src/build-data/makefile/unix_shr.in	99562e4eb73592c98a2e41842e4e5f2342aff7b7
+++ src/build-data/makefile/unix_shr.in	5c2c8418a24ade03d2a66eb5ab677907c246864e
@@ -106,7 +106,7 @@ doxygen:
 shared: $(SHARED_LIB)
 
 doxygen:
-	doxygen %{doc_src_dir}/botan.doxy
+	doxygen %{build_dir}/botan.doxy
 
 clean:
 	$(RM_R) %{build_dir}/lib/* %{build_dir}/checks/*
@@ -114,7 +114,7 @@ distclean: clean
 
 distclean: clean
 	$(RM_R) %{build_dir}
-	$(RM_R) %{doc_src_dir}/doxygen %{doc_src_dir}/botan.doxy
+	$(RM_R) %{doc_src_dir}/doxygen
 	$(RM) Makefile $(CONFIG_SCRIPT) $(PKGCONFIG)
 
 install: $(LIBRARIES)
